### PR TITLE
Remove docker registry from name of image built by buildDockerImage task

### PR DESCRIPTION
**Pull request by driesdeproost**

### CODE REVIEW

**Comment by driesdeproost** [comment]
fixes #24

File: start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/extensions/alfred/telemetry/grafana/GrafanaDockerComposeProjectGenerationConfiguration.java [hunk 1]
```diff
@@ -22,7 +22,7 @@
     DockerComposeCustomizer grafanaDockerComposeCustomizer() {
         return (compose) -> compose.file("grafana").getServices()
                 .service("grafana")
-                .image("hub.xenit.eu/grafana:5.0")
+                .image("hub.xenit.eu/public/grafana:5.0")
```
Comment (tgeens) [inline start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/extensions/alfred/telemetry/grafana/GrafanaDockerComposeProjectGenerationConfiguration.java]: Does this image still exists ? I think we need to create a ticket for that first ?

Comment (driesdeproost) [reply start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/extensions/alfred/telemetry/grafana/GrafanaDockerComposeProjectGenerationConfiguration.java]: nope, my bad
`hub.xenit.eu/public/grafana:0.0.1-5` seems to exist. What about that one?

Comment (tgeens) [reply start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/extensions/alfred/telemetry/grafana/GrafanaDockerComposeProjectGenerationConfiguration.java]: @driesdeproost `5:0` also does not exist
1. you should look that up first
2. you should verify this works

Comment (driesdeproost) [reply start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/extensions/alfred/telemetry/grafana/GrafanaDockerComposeProjectGenerationConfiguration.java]: for point 2 however, there seems to be a bug with the application of our amp plugin in a setup with telemetry and micrometer.

Out of scope for this ticket though

Comment (tgeens) [reply start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/extensions/alfred/telemetry/grafana/GrafanaDockerComposeProjectGenerationConfiguration.java]: Aha :+1:

**Pull request merged**